tests/models/store.py

```diff
@@ -233,8 +236,11 @@ def test_update_set_last_sync_revision(ru_update_set_last_sync_revision_po):
 
 @pytest.mark.django_db
 def test_update_upload_defaults(en_tutorial_po, system):
-    _update_from_upload_file(en_tutorial_po,
-                             "tests/data/po/tutorial/en/tutorial_update.po")
+    update_store(
+        en_tutorial_po,
+        [("Hello, world", "Hello, world UPDATED")],
```

Comment:
@phlax, does `en_tutorial_po` remain a standard fixture? And it contains one `Hello, world` unit. This is a kind of hardcode. Are we looking for a unit for updating by its source string here?